integration/mesos/src/main/java/tachyon/mesos/TachyonFramework.java

```diff
@@ -242,6 +247,8 @@ public void statusUpdate(SchedulerDriver driver, Protos.TaskStatus status) {
         case TASK_ERROR:
           if (status.getTaskId().getValue().equals(String.valueOf(mMasterTaskId))) {
             mMasterCount --;
+          } else if (status.getTaskId().getValue().equals(String.valueOf(mWorkerTaskId))) {
+            mWorkerCount --;
```

Comment:
what does this mean? A worker is lost? We'd better print something. Same as the case of `mMasterCount --`